.buildkite/scripts/steps/publish_oas_docs.sh

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+source .buildkite/scripts/common/util.sh
+
+## Install the bump CLI for just this step
+npm install -g bump-cli@^2.8.2
```

Comment:
Can we also include the package-lock.json?

Reply:
Ah, it was being git ignored, should in now

Reply:
I believe `npm install -g` is going to ignore the lockfile, is npm install sufficient?

Reply:
Fixed since [a2bdc89](https://github.com/elastic/kibana/pull/197482/commits/a2bdc89691c6ad20766a68c421362ed797f83040)